src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Helper\Helper;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpKernel\Kernel;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * A console command to display information about the current installation.
+ *
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+class AboutCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('about')
+            ->setDescription('Displays information about the current installation')
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        /** @var $kernel KernelInterface */
+        $kernel = $this->getContainer()->get('kernel');
+        $baseDir = realpath($kernel->getRootDir().DIRECTORY_SEPARATOR.'..');
+        $bundles = array_map(function ($bundle) use ($baseDir) {
+            return '* '.$bundle->getName().' (<comment>'.self::formatPath($bundle->getPath(), $baseDir).'</>)';
+        }, $kernel->getBundles());
```

Comment:
I would not list the bundles as there are easily accessible in the Kernel class.